[python27] It fails to build python27 on armv7l

try "crew build python27" and get errors:


39 tests failed:
    test_unicode_file test_unicodedata test_univnewlines
    test_univnewlines2k test_unpack test_urllib test_urllib2
    test_urllib2_localnet test_urllib2net test_urllibnet test_urlparse
    test_userdict test_userlist test_userstring test_uu test_uuid
    test_wait3 test_wait4 test_warnings test_wave test_weakref
    test_weakset test_whichdb test_winreg test_winsound test_with
    test_wsgiref test_xdrlib test_xml_etree test_xml_etree_c
    test_xmllib test_xmlrpc test_xpickle test_xrange test_zipfile
    test_zipfile64 test_zipimport test_zipimport_support test_zlib
37 tests skipped:
    test_aepack test_al test_applesingle test_bsddb test_bsddb185
    test_bsddb3 test_cd test_cl test_codecmaps_cn test_codecmaps_hk
    test_codecmaps_jp test_codecmaps_kr test_codecmaps_tw test_curses
    test_dbm test_gdb test_gl test_idle test_imgfile test_kqueue
    test_linuxaudiodev test_macos test_macostools test_msilib test_nis
    test_ossaudiodev test_scriptpackages test_smtpnet
    test_socketserver test_startfile test_sunaudiodev test_tcl
    test_timeout test_tk test_ttk_guionly test_ttk_textonly
    test_turtle
10 skips unexpected on linux2:
    test_bsddb test_bsddb3 test_dbm test_gdb test_idle test_tcl
    test_tk test_ttk_guionly test_ttk_textonly test_turtle

Total duration: 11 min 55 sec
Tests result: FAILURE
Traceback (most recent call last):
  File "./Lib/test/regrtest.py", line 2037, in <module>
  File "./Lib/test/regrtest.py", line 2024, in main_in_temp_cwd
  File "/usr/local/tmp/crew/Python-2.7.14.tar.xz.dir/Python-2.7.14/Lib/contextlib.py", line 35, in __exit__
  File "/usr/local/tmp/crew/Python-2.7.14.tar.xz.dir/Python-2.7.14/Lib/test/support/__init__.py", line 807, in temp_cwd
  File "/usr/local/tmp/crew/Python-2.7.14.tar.xz.dir/Python-2.7.14/Lib/contextlib.py", line 35, in __exit__
  File "/usr/local/tmp/crew/Python-2.7.14.tar.xz.dir/Python-2.7.14/Lib/test/support/__init__.py", line 787, in change_cwd
OSError: [Errno 2] No such file or directory: '/usr/local/tmp/crew/Python-2.7.14.tar.xz.dir/Python-2.7.14'
make: *** [Makefile:869: test] Error 1
python27 failed to build: `TMPDIR=/usr/local/tmp http_proxy= https_proxy= ftp_proxy= make test` exited with 2

--- a/packages/python27.rb
+++ b/packages/python27.rb
@@ -3,22 +3,9 @@ require 'package'
 class Python27 < Package
   description 'Python is a programming language that lets you work quickly and integrate systems more effectively.'
   homepage 'https://www.python.org/'
-  version '2.7.14'
+  version '2.7.14-0'
   source_url 'https://www.python.org/ftp/python/2.7.14/Python-2.7.14.tar.xz'
   source_sha256 '71ffb26e09e78650e424929b2b457b9c912ac216576e6bd9e7d204ed03296a66'
-
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/python27-2.7.14-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/python27-2.7.14-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/python27-2.7.14-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/python27-2.7.14-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: 'ce57af02374678753825f216938d64d98c08f6e1475b92880491efc16b79e9ff',
-     armv7l: 'ce57af02374678753825f216938d64d98c08f6e1475b92880491efc16b79e9ff',
-       i686: 'd1d0449f4bae645a39ef827e03cb892dc701c28115e3ed1404d074df964b4caf',
-     x86_64: 'e446e1313a28c4a75303a0fa108022a7fc73d0c15a50570c8949d194bc44e51b',
-  })
 
   depends_on 'bz2' => :build
   depends_on 'ncurses'

